hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/protocol/dmr/DMRDriver.java

```diff
@@ -126,40 +119,30 @@ public boolean attributeExists(AttributeLocation<DMRNodeLocation> location) {
     public Object fetchAttribute(AttributeLocation<DMRNodeLocation> location) throws ProtocolException {
         String[] attribute = location.getAttribute().split("#");
         String useAttribute = attribute[0];
-        ReadAttributeOperationBuilder<?> opBuilder = OperationBuilder
-                .readAttribute()
-                .address(location.getLocation().getPathAddress())
-                .resolveExpressions()
-                .name(useAttribute);
 
-        // time the execute separately - we want to time ONLY the execute call
-        OperationResult<?> opResult;
-        try (Context timerContext = diagnostics.getDMRRequestTimer().time()) {
-            opResult = opBuilder.execute(client);
```

Comment:
this - this is what I want to time. and just that invocation I want diagnostics on. I want to know the performance of the underlying DMR request.